scripts/vellum-to-hq

```diff
@@ -107,7 +113,10 @@ def build_vellum(path, branch, test=True):
 
 
 def get_old_vellum_rev(path):
-    with open(join(path, "version.txt")) as fh:
+    filename = join(path, "version.txt")
+    if not os.path.isfile(filename):
+        return ""
```

Comment:
What does this do for printing vellum changes? https://github.com/dimagi/commcare-hq/blob/orangejenny-patch-4/scripts/vellum-to-hq#L123 doesn't look like there are any catches to check if this returns a valid commit